### ISSUE TITLE
add more explicit call outs for the floating point numbers changes in .net core 3

### ISSUE THREAD

**Issue by bigworld12** [dotnet/docs#11312]
the article at https://devblogs.microsoft.com/dotnet/floating-point-parsing-and-formatting-improvements-in-net-core-3-0/
gave some nice indicators to the changes happening regarding floating point numbers, but i think they aren't enough, for example users need to know that "-0" is now a valid output of the `.ToString` method (which wasn't stated in the article nor the documentation)

it would also be nice to state all other behaviors of -0
(e.g. `-0.0 == 0.0` is true)

**Comment by rpetrusha** [comment]
Thanks for opening this issue, @bigworld12. We've moved it to the dotnet/docs repo and added it to the work items for the .NET Core 3.0 release.

**Comment by mairaw** [comment]
@rpetrusha were you planning to address this one last one? It wasn't assigned.

**Comment by rpetrusha** [comment]
Yes, I was planning to address this one, @mairaw. Thanks for finding it.

**Comment by ScottFerg56** [comment]
I'm trying to work around this problem in a program migration to .NET 5.0 where the output of "-0" to files is an unacceptable compatibility break and displaying "-0" to an end user is unacceptable behavior. I had hoped that a multiple-section custom numeric format would be a viable, albeit costly, solution. From [Custom numeric format strings](https://docs.microsoft.com/en-us/dotnet/standard/base-types/custom-numeric-format-strings):

> For example, negative values are always displayed without a minus sign when section separators are used.

and

> If the number to be formatted is negative, but becomes zero after rounding according to the format in the second section, the resulting zero is formatted according to the first section.

A code example:
```
// the following displays: value: -0
// should display: value: 0
Debug.WriteLine($"value: {-0.000005:0.####;0.####}");

```
Technically, the number becomes negative zero and NOT zero. So the second claim becomes incomplete at best, at least unsatisfying as to which section applies. But the first claim is irrefutable.

I'm still looking for a satisfying workaround, if you have one. But I thought you should know of this specific issue!
